contributors/devel/api-conventions.md

```diff
@@ -563,6 +566,43 @@ Strategic Merge Patch also supports special operations as listed below.
 
 ### List Operations
 
+#### List of Scalars
+
+We have two merge strategies for list of scalars: replace and merge. Default strategy is replace,
+which will preserve the order; while merge strategy works like an unordered set. We call a list
+using merge strategy a merging list.
```

Comment:
I can give it an alias. I think we should keep the name `merging list`, since it has been used elsewhere. If abandon it, people will feel lost when read this name at other places.